### PR TITLE
Created adding form

### PR DESCRIPTION
**Pull request by Dinalii**

### CODE REVIEW

File: src/component/DiaryHome.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+          underlineShow={true}
+          color="white"
+          id="Title"
+          placeholder="Submit New"
+          value={title}
+          onChange={(e) => setTitle(e.target.value)}
+          inputStyle={{ color: "white", padding: "0 15px" }}
+          style={{ background: "#fed8b1", borderRadius: 25 }}
+        />
+        <br />
```
Comment (Dulajdeshan) [inline src/component/DiaryHome.js]: Use margins and paddings instead of br tags.

File: src/component/DiaryHome.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      </form>
+    </Box>
+  );
+}
+
+const useStyles = makeStyles({
+  root: {
+    "& > *": {
+      padding: "10px 10px 10px 10px",
+      width: "25ch",
```
Comment (Dulajdeshan) [inline src/component/DiaryHome.js]: Use `px` instead `ch`

File: src/component/DiaryHome.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+import TextField from "@material-ui/core/TextField";
+import TextareaAutosize from "@material-ui/core/TextareaAutosize";
+import Button from "@material-ui/core/Button";
+
+export default function DiaryHome() {
+  const classes = useStyles();
+  const [title, setTitle] = useState("");
+  const [description, setDescription] = useState("");
+  const onClick = (e) => {
+    console.log("title=" + title + " description=" + description);
```
Comment (Dulajdeshan) [inline src/component/DiaryHome.js:13]: Remember to remove logs before raise a PR

**Review by Dulajdeshan — CHANGES_REQUESTED** [review]
Go through the comments

File: src/component/DiaryHome.js (hunk 1 above)
Comment (Dinalii) [reply src/component/DiaryHome.js]: Done

File: src/component/DiaryHome.js (hunk 2 above)
Comment (Dinalii) [reply src/component/DiaryHome.js]: Done

**Review by Dulajdeshan — APPROVED** [review]

**Pull request merged**